src/api/vfs/sync_io.rs

```diff
@@ -119,7 +126,14 @@ impl FileSystem for Vfs {
     ) -> Result<(stat64, Duration)> {
         match self.get_real_rootfs(inode)? {
             (Left(fs), idata) => fs.setattr(ctx, idata.ino(), attr, handle, valid),
-            (Right(fs), idata) => fs.setattr(ctx, idata.ino(), attr, handle, valid),
+            (Right(fs), idata) => {
+                let ino = self.convert_inode(idata.fs_idx(), idata.ino())?;
+                fs.setattr(ctx, idata.ino(), attr, handle, valid)
+                    .map(|(mut attr, duration)| {
+                        attr.st_ino = ino;
```

Comment:
`idata` is already an `VfsInode`. Something like
```
attr.st_ino = idata.into();
```
should suffice.

Reply:
Fixed, it works. PTAL